pkg/api/entries.go

```diff
@@ -211,6 +207,23 @@ func CreateLogEntryHandler(params entries.CreateLogEntryParams) middleware.Respo
 	logEntry := models.LogEntry{
 		uuid: logEntryAnon,
 	}
+	return logEntry, http.StatusOK, "", nil
+}
+
+// CreateLogEntryHandler creates new entry into log
+func CreateLogEntryHandler(params entries.CreateLogEntryParams) middleware.Responder {
+	ctx := params.HTTPRequest.Context()
+	httpReq := params.HTTPRequest
+
+	logEntry, code, message, err := createLogEntry(ctx, params)
+	if err != nil {
+		return handleRekorAPIError(params, code, err, message)
```

Comment:
I think I might prefer the order of these params to be the same for both, 
like if it could be `..., code, err, message` to match the inputs to `handleRekorApiError`

Reply:
Maybe this doens't make sense. Feel free to ignore.

Reply:
resolved by way of removing :)